src/libaktualizr/storage/storage_common_test.cc

```diff
@@ -538,9 +530,7 @@ int main(int argc, char **argv) {
   logger_set_threshold(boost::log::trivial::trace);
 
   std::cout << "Running tests for SQLStorage" << std::endl;
-  TemporaryDirectory temp_dir2;
-  storage_test_dir = temp_dir2.Path();
-  storage_test_config = MakeConfig(StorageType::kSqlite, storage_test_dir);
+  current_storage_type = StorageType::kSqlite;
```

Comment:
Temporary directories per test instead of one big one seems like a good idea. Why did we do it the old way? Was it just because of the way we used to support FS and SQL storage?